apluslms_roman/builder.py

```diff
@@ -50,6 +64,8 @@ def __init__(self, backend_class=None, settings=None):
         if backend_class is None:
             if settings and 'backend' in settings:
                 backend_class = settings['backend']
+                if 'docker' in settings and 'directory_map' in settings['docker']:
```

Comment:
This should not be needed. In addition, this is not a very good way to implement this any way. ref: https://github.com/apluslms/roman/pull/8/files#r292131426